docs/docs/configuration/live.md

```diff
@@ -34,11 +41,12 @@ webrtc:
     - stun:8555
 ```
 
-and pass that config to Frigate via docker or `frigate-go2rtc.yaml` for addon users:
+You then need to pass that config to Frigate via docker (see below), or create it as `frigate-go2rtc.yaml` for addon users.
 
-See https://github.com/AlexxIT/go2rtc#module-webrtc for more details
 
 ```yaml
 volumes:
   - /path/to/your/go2rtc.yaml:/config/frigate-go2rtc.yaml:ro
 ```
+
+See https://github.com/AlexxIT/go2rtc#module-webrtc for more details on configuring go2rtc.
```

Comment:
If this is just for configuring go2rtc then we should link to the repo generally and not the webrtc section